### PR TITLE
[Zero Trust] RBAC docs changes

### PR DESCRIPTION
**Pull request by abracchi-tw**

### CODE REVIEW

**Comment by 7michele7** [comment]
👍

File: products/cloudflare-one/src/content/cloudflare-teams-roles-permissions.md [hunk 1]
```diff
@@ -5,30 +5,30 @@ pcx-content-type: reference
...
 
-| Role | Permissions |
-| ---- | ----------- |
-| Super Administrator - All Privileges | <ul><li>Can edit any Cloudflare for Teams setting.</li><li>Can make purchases, update billing, and manage memberships.</li><li>Can revoke the access of other Super Administrators.</li></ul> |
-| Administrator | <ul><li>Can access the full Cloudflare for Teams account except for membership management and billing.</li></ul> |
-| Cloudflare for Teams | <ul><li>Can edit Cloudflare for Teams.</li></ul> |
-| Cloudflare for Teams PII | <ul><li>Can access Cloudflare for Teams PII data.</li></ul> |
-| Cloudflare for Teams Read Only | <ul><li>Can access Cloudflare for Teams in read only mode.</li></ul> |
-| Cloudflare for Teams Reporting | <ul><li>Can access Cloudflare for Teams reporting data.</li></ul> |
+| | Access Read | Access Edit | Gateway Read | Gateway Edit | Gateway Report | Billing Read | Billing Edit |
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/cloudflare-teams-roles-permissions.md:14]: If no user has Billing Edit rights, should it still appear in this table?

File: products/cloudflare-one/src/content/cloudflare-teams-roles-permissions.md [hunk 2]
```diff
@@ -5,30 +5,30 @@ pcx-content-type: reference
 
 # Roles and permissions
 
-When creating a Cloudflare for Teams account, you will be given the Super Administrator role. With this role, you can invite members to join your Teams account, and assign them different roles depending on the permissions you’d like to give them.
+When creating a Cloudflare Zero Trust account, you will be given the Super Administrator role. With this role, you can invite members to join your Zero Trust account, and assign them different roles depending on the permissions you’d like to give them.
 
-A member can be given one of six roles within a Teams account:
+Only Super Administrators of Enterprise accounts will be able to assign or remove the following roles from users in their account. Scroll to the right to see a full list of permissions for each role.
```
Comment (marciocloudflare) [inline products/cloudflare-one/src/content/cloudflare-teams-roles-permissions.md:10]: Why `Scroll to the right`? Is there other content not shown here that makes it necessary to scroll?

**Review by marciocloudflare — APPROVED** [review]

File: products/cloudflare-one/src/content/cloudflare-teams-roles-permissions.md (hunk 2 above)
Comment (abracchi-tw) [reply products/cloudflare-one/src/content/cloudflare-teams-roles-permissions.md:10]: yeah when you look at the table on the page it only shows a few fields (table is too long), and I've had people complain about that before

File: products/cloudflare-one/src/content/cloudflare-teams-roles-permissions.md (hunk 1 above)
Comment (abracchi-tw) [reply products/cloudflare-one/src/content/cloudflare-teams-roles-permissions.md:14]: great catch, I was missing one `x`

**Pull request merged**